src/main/scala/nl.knaw.dans.easy.deposit/docs/dm/Spatial.scala

```diff
@@ -23,45 +25,48 @@ object Spatial {
   val RD_SRS_NAME = "http://www.opengis.net/def/crs/EPSG/0/28992"
 }
 
-trait SchemedSpatial extends Requirements {
-  val scheme: String
-  requireNonEmptyString(scheme)
+trait SchemedSpatial extends Mandatory {
+  val scheme: Option[String]
 
   lazy val srsName: String = {
     scheme match {
-      case "degrees" => Spatial.DEGREES_SRS_NAME
-      case "RD" => Spatial.RD_SRS_NAME
-      case s if s.trim.isEmpty => null // will suppress the XML attribute
-      case _ => scheme
+      case Some("degrees") => Spatial.DEGREES_SRS_NAME
+      case Some("RD") => Spatial.RD_SRS_NAME
+      case Some(s) if s.trim.nonEmpty => s
+      case _ => null // will suppress the XML attribute
     }
   }
-}
 
-case class SpatialPoint(override val scheme: String,
-                        x: String,
-                        y: String,
-                       ) extends Requirements with SchemedSpatial {
-  requireDouble(x)
-  requireDouble(y)
+  private[docs] override def hasMandatory: Boolean = scheme.isProvided
+}
 
+case class SpatialPoint(override val scheme: Option[String],
+                        x: Option[String],
+                        y: Option[String],
+                       ) extends SchemedSpatial {
+  private val sx: String = x.getOrElse("")
+  private val sy: String = y.getOrElse("")
   lazy val pos: String = srsName match {
-    case Spatial.RD_SRS_NAME => s"$x $y"
-    case Spatial.DEGREES_SRS_NAME => s"$y $x"
-    case _ => s"$y $x"
+    case Spatial.RD_SRS_NAME => s"$sx $sy"
+    case Spatial.DEGREES_SRS_NAME => s"$sy $sx"
```

Comment:
There is a chance this might change. I'm doubting whether I should give you the lat/lon in the correct form from the UI, rather than you doing the conversion here...
I'll discuss this with @lindareijnhoudt tomorrow